TimeEvaluate.go

```diff
@@ -0,0 +1,42 @@
+package cmds
+
+import (
+	"fmt"
+	"time"
+)
+
+type TimeEvaluate struct {
+	TmVal   []time.Time
+	TmIndex int
+	Event   []string
+}
+
+func (t *TimeEvaluate) init() {
+	t.TmVal = make([]time.Time, 0)
+	t.TmIndex = 0
+	t.Event = make([]string, 0)
+	t.RecordTime("Start")
+}
+
+func (t *TimeEvaluate) RecordTime(event string) {
+	if t.TmVal == nil {
+		t.init()
+	}
+	t.TmVal = append(t.TmVal, time.Now())
+	t.Event = append(t.Event, event)
+	t.TmIndex++
+}
+func (t *TimeEvaluate) Report() string {
+	if t.TmIndex == 0 {
+		return ""
+	}
+
+	result := "Report(InNanoS):Start"
+	for i := 1; i < t.TmIndex; i++ {
+		costTime := t.TmVal[i].UnixNano() - t.TmVal[i-1].UnixNano()
+		result = fmt.Sprintf("%s-%s:%d", result, t.Event[i], costTime)
```

Comment:
add line breaks between reports.

Also, https://golang.org/pkg/strings/#Builder is more efficient for this type of concatenation